website/content/docs/agent/autoauth/index.mdx

```diff
@@ -138,8 +138,13 @@ These are common configuration values that live within the `method` block:
   structure. Values can be an integer number of seconds or a stringish value
   like `5m`.
 
+- `min_backoff` `(string or integer: "1s")` - The initial backoff time Agent 
```

Comment:
Should be parameter be `min_backoff` or `initial_backoff`? I see mixed references between min and initial even though they refer to the same thing.

Reply:
I changed it to `min_backoff` in https://github.com/hashicorp/vault/pull/15204/commits/baba936a9c75f8ef0454dca7878a8dd4594c64e1 (it was `initial_backoff`) because it aligned better with the existing parameter `max_backoff`.

Reply:
I removed all initial references.